src/v1/cloud_build_client.js

```diff
@@ -0,0 +1,1263 @@
+   *     const response = responses[0];
+   *     // doThingsWith(response)
+   *   })
+   *   .catch(err => {
+   *     console.error(err);
+   *   });
+   */
+  getBuild(request, options, callback) {
+    if (options instanceof Function && callback === undefined) {
+      callback = options;
+      options = {};
+    }
+    request = request || {};
+    options = options || {};
+
+    return this._innerApiCalls.getBuild(request, options, callback);
+  }
+
+  /**
+   * Lists previously requested builds.
+   *
+   * Previously requested builds may still be in-progress, or may have finished
+   * successfully or unsuccessfully.
+   *
+   * @param {Object} request
+   *   The request object that will be sent.
+   * @param {string} request.projectId
+   *   Required. ID of the project.
+   * @param {number} [request.pageSize]
+   *   The maximum number of resources contained in the underlying API
+   *   response. If page streaming is performed per-resource, this
+   *   parameter does not affect the return value. If page streaming is
+   *   performed per-page, this determines the maximum number of
+   *   resources in a page.
+   * @param {string} [request.filter]
+   *   The raw filter text to constrain the results.
+   * @param {Object} [options]
+   *   Optional parameters. You can override the default settings for this call, e.g, timeout,
+   *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/interfaces/CallOptions.html} for the details.
+   * @param {function(?Error, ?Array, ?Object, ?Object)} [callback]
+   *   The function which will be called with the result of the API call.
+   *
+   *   The second parameter to the callback is Array of [Build]{@link google.devtools.cloudbuild.v1.Build}.
+   *
+   *   When autoPaginate: false is specified through options, it contains the result
+   *   in a single response. If the response indicates the next page exists, the third
+   *   parameter is set to be used for the next request object. The fourth parameter keeps
+   *   the raw response object of an object representing [ListBuildsResponse]{@link google.devtools.cloudbuild.v1.ListBuildsResponse}.
+   * @returns {Promise} - The promise which resolves to an array.
+   *   The first element of the array is Array of [Build]{@link google.devtools.cloudbuild.v1.Build}.
+   *
+   *   When autoPaginate: false is specified through options, the array has three elements.
+   *   The first element is Array of [Build]{@link google.devtools.cloudbuild.v1.Build} in a single response.
+   *   The second element is the next request object if the response
+   *   indicates the next page exists, or null. The third element is
+   *   an object representing [ListBuildsResponse]{@link google.devtools.cloudbuild.v1.ListBuildsResponse}.
+   *
+   *   The promise has a method named "cancel" which cancels the ongoing API call.
+   *
+   * @example
+   *
+   * const cloudbuild = require('cloudbuild.v1');
```

Comment:
@wlynch we should add the stanza, `domain_layer_location: google-cloud`, here:

https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudbuild/v1/cloudbuild_gapic.yaml#L18

So that the generated code includes `require('@google-cloud/cloudbuild')`. @alexander-fenster any thoughts on why this generated code is including the `.v1` (this is incorrect, and I don't believe was happening in the past).

```
  nodejs:
    package_name: recaptchaenterprise.v1beta1
    domain_layer_location: google-cloud
```

Reply:
this looks to have been fixed :+1: